app/lib/actions/pulp3/content_view_version/create_import.rb

```diff
@@ -1,22 +1,22 @@
 module Actions
   module Pulp3
     module ContentViewVersion
-      class Import < Pulp3::AbstractAsyncTask
+      class CreateImport < Pulp3::AbstractAsyncTask
```

Comment:
There are some places where the name change isn't propagated, like in the finalize step for [app/lib/actions/katello/content_view_version/import.rb](https://github.com/Katello/katello/pull/9944/files#diff-95d4432c1a79855e7a9c358ab57dcdb08066f32163216c5ac1c4425b1e3fcadd)

Reply:
Ah missed that in the rebase I did just before pushing. I think that was the only one? Fixed now

Reply:
Looks like that was it!